server/CMakeLists.txt

```diff
@@ -1,22 +1,31 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 2.6)
```

Comment:
Why downgrade?

Reply:
Actually not a necessary downgrade. A Forgotten debug trail...